apps/crisis-unleashed/components/logo-system/logo-variant.tsx

```
@@ -1,163 +1,429 @@
+        return renderVerticalVariant(
+          faction, 
+          primaryColor, 
+          secondaryColor, 
+          pixelSize, 
+          showTagline ? tagline : null
+        );
+      
+      case "minimal":
+        return renderMinimalVariant(faction, primaryColor, secondaryColor, pixelSize);
+      
+      case "standard":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)